tokens/properties/alias.json

```
@@ -433,44 +433,11 @@
     "transition": {
         "interaction": {
             "button": {
-                "hover": {
-                    "property": {
-                        "value": "{transition.property.background-color.value}, {transition.property.border-color.value}"
-                    },
-                    "duration": {
-                        "value": "{transition.duration.fast.value}"
-                    }
+                "property": {
+                    "value": "{transition.property.background-color.value}, {transition.property.background-color.value}, {transition.property.border-color.value}, {transition.property.box-shadow.value}"
```

The text color of neutral buttons is lightened on hover (see deleted line 461), so the token {transition.property.color.value} should also be added here. 
```suggestion
                    "value": "{transition.property.background-color.value}, {transition.property.border-color.value}, {transition.property.box-shadow.value}, {transition.property.color.value} "
```

---

Yup, the duplicated `background-color` was totally my failed attempt at adding the `color` property. I blame the heat. Thanks for spotting that! Fixed!